uwblueprint/supportive-housing
# get residents and employees from db

## GitHub Issue link
https://github.com/uwblueprint/supportive-housing/issues/87


## Implementation description
* use API to get employees
* have employees from db in the filter dropdown for employees and attention tos
* use API to get residents
* have residents from db in the filter dropdown for residents 



## Steps to test
1. In the filter, check if all the users in the db are included in the select options for Employee
2. Check if the filtering works for Employee
3. Repeat steps 1 + 2 for Attention To 
4. In the filter check is all residents in the db are included in the select option for Resident
5. Check if the filtering works for residents



## What should reviewers focus on?
* 


## Checklist
